source/reference/api/start.txt

```diff
@@ -362,23 +362,20 @@ If the ``start`` request is successful, ``mongosync`` enters the
 Pre-Split Chunks
 ~~~~~~~~~~~~~~~~
 
-.. versionadded:: 1.1
-
-When ``mongosync`` syncs to a sharded cluster, it pre-splits chunks for
-sharded collections on the destination cluster.  This is supported in the
-following configurations:
+When ``mongosync`` syncs to a sharded destination cluster, it pre-splits chunks 
+for sharded collections on the destination cluster. For each sharded collection, 
+``mongosync`` creates twice as many chunks as there are shards in the 
+destination cluster. This is supported in the following configurations:
```

Comment:
small suggestion for active voice (just spitballing, not sure if "procedure" is the right word here)

```suggestion
destination cluster. The following configurations support this procedure:
```

Reply:
Makes sense! Updating.